resources/lib/playerinfo.py

```diff
@@ -201,20 +202,35 @@ def push_position(self, position=0, total=100):
         if not self.asset_id:
             return
 
+        watch_later = None
+        # Disable watch_later when completely watched
+        if self.resumepoints.is_watchlater(self.asset_id) and position > total - SECONDS_MARGIN:
+            watch_later = False
+
         # Push resumepoint to VRT NU
         self.resumepoints.update(
             asset_id=self.asset_id,
             title=self.title,
             url=self.url,
+            watch_later=watch_later,
             position=position,
             total=total,
             whatson_id=self.whatson_id,
             asynchronous=True
         )
 
-        # Do not reload container and rely on Kodi internal watch status, when watching a single episode.
+        # Kodi uses different resumepoint margins than VRT NU, to obey to VRT NU resumepoint margins we need to reload the container
+        # to overwrite Kodi watch status when the position is in these margins.
+        # Use setting from advancedsettings.xml or default value
+        # https://github.com/xbmc/xbmc/blob/master/xbmc/settings/AdvancedSettings.cpp
+        # https://kodi.wiki/view/HOW-TO:Modify_automatic_watch_and_resume_points
+        ignoresecondsatstart = get_advanced_setting('video/ignoresecondsatstart') or 180
+        ignorepercentatend = get_advanced_setting('video/ignorepercentatend') or 8
```

Comment:
Beware that if people have configured this to be 0, it would go to the default.

It is probably better to have this interface: `def get_advanced_setting(key, default=None):`
And then call it as:
```python
        ignoresecondsatstart = get_advanced_setting('video/ignoresecondsatstart', default=180)
        ignorepercentatend = get_advanced_setting('video/ignorepercentatend', default=8)
```

Reply:
Fixed.